src/test/java/com/Automation/APIAutomation/UserManagement/postUsers.java

```diff
@@ -82,7 +77,7 @@ public void validatePutWithString() {
                 .body("{\"name\":\"pallavisawale\",\"job\":\"SDET\"}")
                 .when()
                 .put("https://reqres.in/api/users/2");
-        AssertHelper.assertEquals(response.getStatusCode(), StatusCode.SUCCESS.code,"");
+        AssertHelper.assertEquals(response.getStatusCode(), StatusCode.SUCCESS.code, "");
```

Comment:
assertions should contains:
1. Status code
2. Response body
3. Response header if required

Reply:
Added validation using response